tasks/tests/test_prime.py

```diff
@@ -257,14 +260,16 @@ class PrimeSessionStorage2(PrimeDataStorageMixin):
         session_storage1 = PrimeSessionStorage1()
         session_storage2 = PrimeSessionStorage2()
 
-        session_storage1.set_default_mto_ids(test_moves)
+        for move in test_moves:
+            session_storage1.set_default_mto_ids(move["id"])
 
         assert session_storage1.default_mto_ids == expected_ids
         assert session_storage1.default_mto_ids == session_storage2.default_mto_ids
         assert len(responses.calls) == 4
```

Comment:
My understanding here, looking at the test moves, is that none have all of the mto_ids, which is why it still makes 4 requests?

I guess this isn't true in practice now with our single move but it probably still tests the method more thoroughly.

Reply:
Yeah with the change I made we likely won't and shouldn't need all 4 requests since our setup should make it so that all of the info is available in the first move. I kept this there as this behavior is still supported in the case that we run the on start function multiple times and for w/e reason there are some moves that don't properly create moves.